default.xml

```diff
@@ -2,12 +2,10 @@
 
 <manifest>
     <remote name="prpl" fetch="https://github.com/prplfoundation/" pushurl="ssh://git@github.com/prplfoundation" />
-    <remote name="boardfarm" fetch="https://github.com/mattsm/" />
+    <remote name="boardfarm" fetch="https://github.com/prplfoundation/" pushurl="ssh://git@github.com/prplfoundation" />
 
     <default revision="master" remote="prpl" sync-j="4" />
 
-    <project path="prplMesh" name="prplMesh"/>
-
-    <!-- upstream boardfarm -->
-    <project path="boardfarm" remote="boardfarm" name="boardfarm" revision="228c3434fe12d596d740a6ab6635dd5f44fa2ae7" />
+    <project path="prplMesh" name="prplMesh" />
+    <project path="boardfarm" name="boardfarm" />
```

Comment:
With this, you don't actually use the `boardfarm` remote defined above, but instead you use the default `prpl` remote. Which is fine, actually. So instead, I would keep the `boardfarm` remote unchanged and unused, so it is still there when we want to switch back to upstream boardfarm.